gridt/src/app/settings/change-email/change-email.page.html

```diff
@@ -0,0 +1,59 @@
+<ion-content class="ion-padding">
```

Comment:
I have not been able to review this work, as I could not load the server response on my device.